src/components/login/LoginForm.jsx

```diff
@@ -5,18 +5,59 @@ import {ActivityIndicator, StyleSheet, Text, TextInput, TouchableOpacity, View}
 import {useDispatch} from 'react-redux';
 import {initSendbird} from '../../redux/slices/sendbird';
 
-import {APP_ID, COLORS, TOKEN, USER_ID} from '../../constants';
+import {COLORS} from '../../constants';
 
 export default function LoginForm() {
   const dispatch = useDispatch();
   const navigation = useNavigation();
   const [state, setState] = useState({
-    appId: APP_ID,
-    userId: USER_ID,
-    token: TOKEN,
+    appId: '',
+    userId: '',
+    token: '',
+    channelUrl: '',
     isLoading: false,
+    hasError: false,
+    errorMessage: '',
   });
 
+  useEffect(() => {
+    AsyncStorage.getItem('loginInformation').then(data => {
+      data = JSON.parse(data);
+      setState({
+        appId: data?.appId || '',
+        userId: data?.userId || '',
+        token: data?.token || '',
+        channelUrl: data?.channelUrl || '',
+        isSignedIn: data?.isSignedIn || false,
+      });
+    });
+  }, []);
+
+  useEffect(() => {
+    if (state.isSignedIn) {
+      setState({
+        ...state,
+        isLoading: true,
+      });
+      dispatch(initSendbird(state))
```

Comment:
In the initSendbird slice, SendbirdChat.init always returns the same instance even if the app ID has been changed, so we should add the newInstance: true option to the SendbirdChat.init parameters:

```ts
sb = SendbirdChat.init({ ..., newInstance: sb?.appId !== data.appId });
```